### PR TITLE
Add roles rw neo4j

### PR DESCRIPTION
**Pull request by scott-ace-newton**

### CODE REVIEW

File: roles-rw-neo4j@.service [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+KillMode=none
+ExecStartPre=-/bin/bash -c '/usr/bin/docker kill %p-%i > /dev/null 2>&1'
+ExecStartPre=-/bin/bash -c '/usr/bin/docker rm %p-%i > /dev/null 2>&1'
+ExecStartPre=/bin/bash -c '/usr/bin/docker history coco/roles-rw-neo4j:$DOCKER_APP_VERSION > /dev/null 2>&1 || /usr/bin/docker pull coco/roles-rw-neo4j:$DOCKER_APP_VERSION'
+
+ExecStart=/bin/sh -c '\
+  export APP_PORT=8080; \
+  export ENV=$(/usr/bin/etcdctl get /ft/config/environment_tag); \
+  export NEO_HOST_PORT=$(/usr/bin/etcdctl get /ft/services/neo4j/servers/1); \
+  /usr/bin/docker run --rm --name %p-%i -p $APP_PORT --env "NEO_URL=$NEO_HOST_PORT/db/data" --env "APP_PORT=$APP_PORT" --env "BATCH_SIZE=50" --env "GRAPHITE_ADDRESS=graphite.ft.com:2003" --env "GRAPHITE_PREFIX=coco.services.$ENV.roles-rw-neo4j.%i" coco/roles-rw-neo4j:$DOCKER_APP_VERSION;'
```
Comment (izzyblues) [inline roles-rw-neo4j@.service]: This is a minor comment. To improve readability, I suggest to format this command as follows:
```
/usr/bin/docker run --rm --name %p-%i -p $APP_PORT \ 
--env "NEO_URL=$NEO_HOST_PORT/db/data" \
--env "APP_PORT=$APP_PORT" \
--env "BATCH_SIZE=50" \
--env "GRAPHITE_ADDRESS=graphite.ft.com:2003" \
--env "GRAPHITE_PREFIX=coco.services.$ENV.roles-rw-neo4j.%i" \
coco/roles-rw-neo4j:$DOCKER_APP_VERSION;'
```

Comment (Zapadlo) [inline roles-rw-neo4j@.service]: Similarly `--env` can be written as `-e` and simple arguments don't have to be double quoted

Comment (scott-ace-newton) [inline roles-rw-neo4j@.service]: None of the other services that I can see adhere to this formatting? I agree that its easier to read but would just introduce inconsistancies between service files...

Comment (Zapadlo) [inline roles-rw-neo4j@.service]: https://github.com/Financial-Times/up-service-files/blob/master/aggregate-healthcheck%40.service
https://github.com/Financial-Times/up-service-files/blob/master/content-ingester-neo4j-red%40.service
https://github.com/Financial-Times/up-service-files/blob/master/content-ingester%40.service

There are a few

File: roles-rw-neo4j@.service [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+ExecStartPre=/bin/bash -c '/usr/bin/docker history coco/roles-rw-neo4j:$DOCKER_APP_VERSION > /dev/null 2>&1 || /usr/bin/docker pull coco/roles-rw-neo4j:$DOCKER_APP_VERSION'
+
+ExecStart=/bin/sh -c '\
+  export APP_PORT=8080; \
+  export ENV=$(/usr/bin/etcdctl get /ft/config/environment_tag); \
+  export NEO_HOST_PORT=$(/usr/bin/etcdctl get /ft/services/neo4j/servers/1); \
+  /usr/bin/docker run --rm --name %p-%i -p $APP_PORT 
+    --env "NEO_URL=$NEO_HOST_PORT/db/data" 
+    --env "APP_PORT=$APP_PORT" 
+    --env "BATCH_SIZE=50"
```
Comment (martingartonft) [inline roles-rw-neo4j@.service]: 50?

File: roles-rw-neo4j-red@.service [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+ExecStartPre=-/bin/bash -c '/usr/bin/docker rm %p-%i > /dev/null 2>&1'
+ExecStartPre=/bin/bash -c '/usr/bin/docker history coco/roles-rw-neo4j:$DOCKER_APP_VERSION > /dev/null 2>&1 || /usr/bin/docker pull coco/roles-rw-neo4j:$DOCKER_APP_VERSION'
+
+ExecStart=/bin/sh -c '\
+  export APP_PORT=8080; \
+  export ENV=$(/usr/bin/etcdctl get /ft/config/environment_tag); \
+  /usr/bin/docker run --rm --name %p-%i -p $APP_PORT 
+    --env "NEO_URL=http://$HOSTNAME:8080/__neo4j-red/db/data" 
+    --env "APP_PORT=$APP_PORT" 
+    --env "BATCH_SIZE=50"
```
Comment (martingartonft) [inline roles-rw-neo4j-red@.service]: 50?

Comment (scott-ace-newton) [inline roles-rw-neo4j-red@.service]: fixed

File: roles-rw-neo4j@.service (hunk 1 above)
Comment (scott-ace-newton) [inline roles-rw-neo4j@.service]: ive left this as I am now blocking catalin

File: roles-rw-neo4j@.service (hunk 2 above)
Comment (scott-ace-newton) [inline roles-rw-neo4j@.service]: fixed

**Comment by catalin-ilea** [comment]
lgtm

**Pull request merged**